ocrd_tool.schema.yml

```
@@ -126,3 +126,52 @@ properties:
                 - layout/segmentation/word
                 - layout/segmentation/classification
                 - layout/analysis
+          resource_locations:
+            type: array
+            description: The locations in the filesystem this processor supports for resource lookup
+            default: ['data', 'cwd', 'system']
+            items:
+              type: string
+              enum: ['data', 'cwd', 'system']
```

Even if we would not need to **register** these concretely under `resources` in the tool json, the processor should specify under `resource_locations` whether it allows for (or provides) any pre-installed resources – **module** location:
```suggestion
          resource_locations:
            type: array
            description: The locations in the filesystem this processor supports for resource lookup
            default: ['data', 'cwd', 'system', 'module']
            items:
              type: string
              enum: ['data', 'cwd', 'system', 'module']
```